var/spack/repos/builtin/packages/glew/package.py

```diff
@@ -15,27 +15,32 @@ class Glew(CMakePackage):
     url = "https://github.com/nigels-com/glew/releases/download/glew-2.1.0/glew-2.1.0.tgz"
     root_cmakelists_dir = "build/cmake"
 
+    maintainers("biddisco")
+
     version("2.2.0", sha256="d4fc82893cfb00109578d0a1a2337fb8ca335b3ceccf97b97e5cc7f08e4353e1")
     version("2.1.0", sha256="04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95")
     version("2.0.0", sha256="c572c30a4e64689c342ba1624130ac98936d7af90c3103f9ce12b8a0c5736764")
 
     variant(
         "gl",
         default="glx" if sys.platform.startswith("linux") else "other",
-        values=("glx", "osmesa", "other"),
+        values=("glx", "osmesa", "egl", "other"),
         multi=False,
         description="The OpenGL provider to use",
     )
     conflicts("^osmesa", when="gl=glx")
+    conflicts("^osmesa", when="gl=egl")
     conflicts("^osmesa", when="gl=other")
     conflicts("^glx", when="gl=osmesa")
     conflicts("^glx", when="gl=other")
+    conflicts("^glx", when="gl=egl")
```

Comment:
I think this also needs guards for `conflicts("^egl", when="gl=<not egl>")`for completeness.